package.json

```diff
@@ -42,6 +42,7 @@
     "not IE 11"
   ],
   "dependencies": {
+    "@axe-core/playwright": "^4.9.1",
```

Comment:
could you check if this can be a `devDependency`?

Reply:
Thank you, yes, I moved both to devDependencies.